Documentation/email-configuration.md

```diff
@@ -20,6 +20,7 @@ set `auth` to `plain` and specify your username and password.
     "host": "smtp.example.org",
     "port": 587,
     "auth": "plain",
```

Comment:
While you're in here can you make this field not required. Currently if it's omitted Dex explodes, but we only support 'plain' and a default. Let's just default to the default if it's not provided.